### ISSUE TITLE
[Search Results] Web Resources should be saved and need to display in my plan page.

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#177]
_From @turpu-rakesh on July 12, 2018 14:52_

#221  Save Resources to user profile.

_Copied from original issue: Microsoft/Access2Justice#229_

**Comment by asaperski-gloriumtech** [comment]
Saving works only if text "Save" is clicked - Save button isn't stretched
Share button has the same issue

**Comment by svasylenko** [comment]
Issue is not resolved, @KellyeDC please set a priority for fixing.
the same wissue with Save and Share:

![image](https://user-images.githubusercontent.com/46557859/51392198-fa218800-1b3c-11e9-9601-6e4433598059.png)

**Comment by yfzwinnie** [comment]
Completed during MVP and closed by Amitabh

**Comment by svasylenko** [comment]
Item is still not fixed.

**Comment by breeglej** [comment]
@KellyeDC @grawdon - There still appear to be issues.  I tested the following after logging in:
1. Save - Nothing happens
2. Share - send link - I could enter my email address and it appeared to do something but no email arrived in my inbox with the link.  I tried both my lsc and gmail emails.
3. Share - copy link - I successfully copied the link and pasted it into a new browser, but the page continued to load for a very long time and never actually successfully loaded.
